apis/task/tasks.py

```diff
@@ -45,7 +46,10 @@ def post(self):
 
     def start_task(self, tx_hash):
         uid = uuid()
-        wait_for_mining.s(tx_hash).apply_async(task_id=uid)
+        try:
+            wait_for_mining.s(tx_hash).apply_async(task_id=uid)
+        except Exception as e:
```

Comment:
We should log the exception out here as this may trigger for something other than a timeout

Reply:
Nevermind, I see it being logged out in the function itself